app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  include Visible
+  belongs_to :article
+
+  VALID_STATUSES = ['public', 'private', 'archived']
```

Comment:
Use `%w` notation for arrays of strings.

Reply:
No, because in the Concern file this operation is already done, and if i remove it, model continues works fine.